src/quilla/__init__.py

```diff
@@ -77,10 +76,29 @@ def make_parser() -> argparse.ArgumentParser:  # pragma: no cover
         '-u',
         '--update-baseline',
         dest='update_baseline',
-        action='store_true',
-        help='Used to update the baseline images for VisualParity. '
+        action='append',
+        metavar='BASELINE_ID',
+        help='Used to update the specific baseline image associated with '
+        'the provided baseline ID for VisualParity. '
         'Different plugins define different behaviour, so this is not '
-        'necessarily a lossless operation.'
+        'necessarily a lossless operation.',
+        default=[],
+    )
+    config_group.add_argument(
+        '-U',
+        '--update-all-baselines',
+        dest='update_all_baselines',
+        action='store_true',
+        help='Used to update all baseline images in a Quilla test file. '
+        'this is equivalent to using --update-baseline with every baseline ID '
+        'that is used in the Quilla test.'
+    )
+    config_group.add_argument(
+        '--create-baseline-if-none',
+        dest='create_baseline_if_none',
+        action='store_true',
+        help='A flag to set that will create a new baseline image if a storage '
+        'mechanism is configured but no baseline image is found with the specified ID.'
```

Comment:
why there is no metavar='BASELINE_ID' for this argument?

Reply:
This one stores true so that any test that includes a missing baseline will generate one, and not be specific to a given test

Reply:
Should we name it to be "create_all_baselines_when_absent" since it's possible that it will create multiple baselines?

Reply:
I think keeping "create-baselines-if-absent" with a clearer help string would be better